ClientApp/src/app/server-api/server-api.service.ts

```diff
@@ -75,21 +75,69 @@ export class ServerApi {
         id: 10000,
         name: "Work on angular",
         assignee: "Sebastian Hardin",
-        storyPoints: 5
+        storyPoints: 5,
+        description: 'This is the description of the task',
+        progress: 'In Review',
+        startDate: new Date(Date.parse('12/27/2022')),
+        endDate: new Date(Date.parse('12/28/2022')),
       };
     } else if (id === 10001) {
       return {
         id: 10001,
         name: "Add properties to database",
         assignee: "Sebastian Hardin",
-        storyPoints: 3
+        storyPoints: 3,
+        description: 'This is the description of the task',
+        progress: 'Backlog',
+        startDate: new Date(Date.parse('12/23/2022')),
+        endDate: new Date(Date.parse('12/26/2022')),
       };
     }
     return {
       id: 0,
       name: "Invalid task",
       assignee: "No one",
-      storyPoints: 0
+      storyPoints: 0,
+      description: 'This is the description of the task',
+      progress: 'Backlog',
+      startDate: new Date(Date.parse('12/23/2022')),
+      endDate: new Date(Date.parse('12/26/2022')),
+    };
+  }
+
+  getFullTaskData(id: number): TaskData {
+    if (id === 10000) {
+      return {
+        id: 10000,
+        name: "Work on angular",
+        assignee: "Sebastian Hardin",
+        storyPoints: 5,
+        description: 'This is the description of the task',
+        progress: 'In Review',
+        startDate: new Date(Date.parse('12/27/2022')),
+        endDate: new Date(Date.parse('12/28/2022')),
+      };
+    } else if (id === 10001) {
+      return {
+        id: 10001,
+        name: "Add properties to database",
+        assignee: "Sebastian Hardin",
+        storyPoints: 3,
+        description: 'This is the description of the task',
+        progress: 'Backlog',
+        startDate: new Date(Date.parse('12/23/2022')),
+        endDate: new Date(Date.parse('12/26/2022')),
+      };
+    }
+    return {
```

Comment:
Is it possible to just have the service throw an error if there's no task with a corresponding ID, and then have the task page display a 404 message? I think that would be a cleaner implementation than either showing default values to the user or having to interpret these default values to know to display a 404 message

Reply:
Yup, that's definitely the end goal. I just have it like this as a placeholder until we're ready to properly digest the backend api.